tests/Magento_Page_Loading.php

```diff
@@ -0,0 +1,473 @@
+<?php
+
+require_once 'vendor/autoload.php';
+
+class MagentoPageLoading extends Sauce\Sausage\WebDriverTestCase
+{
+    protected $base_url = 'http://127.0.0.1/';
+
+    public static $browsers = [
+        [
+            'browserName' => 'chrome',
+            'desiredCapabilities' => [
+                'version' => '45.0',
+                'platform' => 'OS X 10.10',
+            ],
+        ],
+    ];
+
+    const ADMIN_USERNAME    = 'denis';
+    const ADMIN_PASSWORD    = 'zavy123';
+    const ORDERS_COUNT      = 50;
+
+    const PAGE_TYPE_ADMIN       = 'admin';
+    const PAGE_TYPE_ORDER       = 'order';
+    const PAGE_TYPE_FRONT       = 'front';
+    const PAGE_TYPE_CUSTOMER    = 'customer';
+
+    private static $frontFailList       = [];
+    private static $customerFailList    = [];
+    private static $adminFailList       = [];
+    private static $orderFailList       = [];
+
+    /**
+     * Pages that don't use magento layout
+     *
+     * @var array
+     */
+    private $customLayoutPages = [
+        [
+            'link' => '/doneright/enhanced/dashboard',
+        ],
+        [
+            'link' => '/doneright/extension_local',
+        ],
+        [
+            'link' => '/doneright/adminhtml_subscribe',
+        ],
+        [
+            'link' => '/doneright/enhanced_settings',
+        ],
+        [
+            'link' => '/doneright/adminhtml_recurring/processDaily',
+        ],
+        [
+            'link' => '/doneright/adminhtml_cryozonic/check',
+        ],
+    ];
+
+    /**
+     * Frontend links
+     *
+     * @var array
+     */
+    private $frontendLinks = [
+        0 => '/the-science-research',
+        1 => '/testimonials',
+        2 => '/faq',
+        3 => '/blog',
+        4 => '/ingredients',
+        5 => '/relieffactor-quickstart-pack.html',
+    ];
```

Comment:
it is better to use dataProvider
see: https://phpunit.de/manual/current/en/writing-tests-for-phpunit.html
   chanpter: 'Data Providers'